README.Rmd

```diff
@@ -26,7 +26,7 @@ The goal of this project is to bring the blazingly fast
 computation engine is written in Rust and this R implementation has no other
 dependencies than R itself (≥ 4.1.0).
 
-Documentation can be found on the **rpolars**
+Documentation can be found on the **r-polars**
```

Comment:
Perhaps personal preference but I think the link link should be to the actual index page (i.e., https://rpolars.github.io/index.html) rather than the reference manual. That way you readers are able top navigate more easily to where they want (e.g., installation section of the README/landing page or the vignette).

Reply:
I will change back to entry page. It is also much prettier now, after latest release, as the main page is now your readme contribution :)

https://rpolars.github.io/